src/nuclio/functions/version/version-configuration/tabs/version-configuration-annotations/version-configuration-annotations.component.js

```diff
@@ -38,7 +38,14 @@
             name: $i18next.t('functions:TOOLTIP.ANNOTATION', {lng: lng})
         });
         ctrl.validationRules = {
-            key: []
+            key: ValidationService.getValidationRules('k8s.prefixedQualifiedName')
+                .concat([
```

Comment:
Instead of this `.concat()` you can just pass the additional rules as second argument to `ValidationService.getValidationRules`. See https://github.com/iguazio/dashboard-controls/blob/development/src/igz_controls/services/validating-patterns.service.js#L354